yesod-bin/Keter.hs

```diff
@@ -79,8 +79,8 @@ keter cabal noBuild noCopyTo buildArgs = do
 
     unless noBuild $ do
         stackQueryRunSuccess <- do
-            (ec,_,_) <- readProcessWithExitCode "stack" ["query"] ""
-            return (ec == ExitSuccess)
+            eres <- try $ readProcessWithExitCode "stack" ["query"] "" :: IO (Either SomeException (ExitCode, String, String))
```

Comment:
Catching all exceptions (via `SomeException`) is dangerous, as it can swallow up async exceptions. Does catching just `IOException` fix this?